### PR TITLE
Adds mining pills, an alternative to the medipen, that allows you to buy heals in bulk!

### PR DESCRIPTION
**Pull request by SgtxRootbeer**
:cl: christy
add: mining pill bottles, contains 20 units of Bicaridine and Kelotane. An alternative to the medipen that allows you to buy your heals in bulk!
/:cl:
babby's first code-related PR to /tg/, be gentle.
also, to stop the balance discussion before it begins, there are both pros and cons.
+relatively strong heals
+useful even to slimepeople
+per-pill, its cheaper then a medipen.
-you can OD on it
-its slower then a medipen
-you have to take off your mask to eat it
-you have to buy it in bulk, you may not use all of the pills you buy.
-vs going to chemistry/asking cargo for first-aid packs, its much more expensive.

### CODE REVIEW

**Comment by Kromgar** [comment]
Medipens aren't in bulk do any of you know how mining works?

**Comment by optimumtact** [comment]
They never said medipens were in bulk, do you know how to read?

**Comment by SgtxRootbeer** [comment]
hmmm, travis failed, imma look into that
give me a few minutes.

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 		new /datum/data/mining_equipment("GAR scanners",		/obj/item/clothing/glasses/meson/gar,									500),
 		new /datum/data/mining_equipment("Explorer's Webbing",	/obj/item/weapon/storage/belt/mining,									500),
 		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			500),
+		new /datum/data/nining_equipment("Brute and burn pill bottle", /obj/item/weapon/storage/pill_bottle/mining_pills				1000),
```
Comment (optimumtact) [inline code/modules/mining/machine_vending.dm]: missing a , after the type path on this line

**Comment by SgtxRootbeer** [comment]
that was fun.

**Comment by Mindustry** [comment]
Taking your mask off isn't really a con because miners are already doing that for cactus and and oxyloss is the easiest of damages to fix.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -149,3 +149,10 @@
 	icon_state = "pill18"
 	list_reagents = list("insulin" = 50)
 	roundstart = 1
+
+/obj/item/weapon/reagent_containers/pill/mining
+	name = "Basic brute and burn pill"
+	desc = "Contains 20 units of bicaridine and kelotane, for a cheap way to mend those lacerations!"
+	icon_state = "pill20"
+	list_reagents = list("Bicaridine" = 20, "Kelotane" = 20)
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/pill.dm]: This is incredibly strong; it'll heal a total of 100 brute and 100 burn damage for a single pill.

**Comment by jughu** [comment]
Ufif the pills were like 3 bica pills 3 kelotane pills amd maybe 1 salbu it be more nalanced then this

File: code/modules/reagents/reagent_containers/pill.dm (hunk 2 above)
Comment (SgtxRootbeer) [inline code/modules/reagents/reagent_containers/pill.dm]: Yes, its meant as a simple alternative to the normal medipen. which, mind you has the following regents
	list_reagents = list("salbutamol" = 10, "leporazine" = 15, "tricordrazine" = 15, "epinephrine" = 10, "miningnanites" = 2, "omnizine" = 5)
the normal magic pen will stabilize your body temp, heal you of some toxin, stop you from taking oxyloss, heal oxyloss, and heal you of 100 dmg, all over the course of about... 5 seconds, give or take. all for 500 pts.
this is about 150 pts per pill, but your healing takes way longer, you dont get oxyloss protection, nor temp stabilization, nor toxin healing.

**Comment by SgtxRootbeer** [comment]
@jughu 
then we might as well use the first aid kits, eh. the idea is that a miner can mindlessly pop this after taking heavy damage in a battle, without having to address specific wounds. also, we already have something like what you described in the code, except its patches, and you get 3 of each. look in the file where i defined my pill bottle for it.
maybe you could make a pr to re-add it.

**Comment by jughu** [comment]
First aid kits take time to applies and doesnt give passive heing while pills are instant take

**Comment by jughu** [comment]
@sgtxrootbeer then why add pills when we litteraly have a box of patches doing both damage types everything shouldnt be easy as piss as miners if they want pills or other things they can ask chemistry jesus christ everyone is removing department cooperation

**Comment by SgtxRootbeer** [comment]
@jughu 
because, i like pills. that and i like general-use pills instead of rummaging through my bag for my last burn patch, only to realize, that i ran out and all i have left is brute patches.
also, about first aid kits, there is one, the brute first-aid kit, costing about 600 pts, 100 pts more then the medipen. mind you, this is a more general-purpose, more expensive, OD-able _alternative_.

**Comment by jughu** [comment]
Patches are nerfed to the ground since syptic powder instantly applies half of the reagents on touch healing you instantly but also passivly they just take time to put on now and the brute kit is stupid you can litteraly order from cargo

**Comment by SgtxRootbeer** [comment]
@jughu if it will make you happy, i can also remove the brute kit seeing as its replaced by cargo interaction. something that isn't too hard for miners to do. although, when i do chemistry, its relatively tedious to make a bottle of pills like i have being sold from the vendor, due to how obtuse the chemmaster and chem dispenser is to use.

**Comment by jughu** [comment]
@sgtxrootbeer chemmaster isnt tedious to use at all just get a chembag a pill bottle a beaker of said reagents put in multiple pills Do your math so it splits evenly the right amount and bam! you got your things

Back to the cargo thing fun fact 1 plasma sheet is 500 points x2=1 medkit/bruise/burn/sufcocation crate since they cost 1000 points you get 3 medkits from one crate plasma doesnt even go for around... 100 points in the orm so its ridicilous overpriced

**Comment by SgtxRootbeer** [comment]
@jughu but to make a pill, that has at least 20 of bicaridine and kelotane, i need to use 1 large beaker to make 2 pills. to make 7, or 10 pills, thats tedious.
thanks for letting me know about those cargo first-aid kits, that helps me justify the price of this even more.

**Comment by jughu** [comment]
@sgtxrootbeer again miners wouldnt buy this even for 1000 points when they can litteraly get patches for a cheap price or the same pills from chemistry with some waiting time

**Comment by SgtxRootbeer** [comment]
@jughu i'd buy it, mainly due to convenience. i dont know about sybil too much, but basil has suffered recently from some relatively short round-times, at around 30 minutes-45 minutes. this is to allow miners, who go and actually get ore, to get the heals they want, within a reasonable time frame. similar to the medipen, but it works differently. 90% of the stuff i pr for anything i make, typically is because i see myself using it in a fun way. maybe im autistic, but i would buy this in a heartbeat instead of the medipen.

**Comment by jughu** [comment]
@sgtxrootbeer use Salicyclic Acid and Oxandrolone since it heals 4 a tick if you have more then 50 burn/brute and if less 0.5 a tick

**Comment by SgtxRootbeer** [comment]
@jughu finally, a good suggestion on how to improve this.

**Comment by jughu** [comment]
@sgtxrootbeer might also add 10 units of Ephedrine so you have stun reducement(isnt that big anyways like 1-2 sec and a increased movement speed so i can quickly take one when tentacled by a goliath and quickly get away from it its 1000 points anyways gotta make it count and it be handy when you take major damage from megafauna

**Comment by SgtxRootbeer** [comment]
I smell power creep with that idea, besides, we have a stimpack kit already.

**Comment by jughu** [comment]
@sgtxrootbeer what about making these into patches with 0 delay. Nanotrasen making pils that requires you to close you off your oxygen supply just to take it sounds counterproductive

**Comment by Militaires** [comment]
i hate this trend of giving miners everything they need roundstart so that the only people miners need to talk to are other miners and scientists.

hey mr chemist give me some heal pills then he sneaks some drugs there and you die due to meth overdose will never happen now

**Comment by jughu** [comment]
@militaires i agree 
Like is it so hard to walk to chemistry ask a pills and then walk away again
Plus 2 plasma sheets equals 4 medkits of any damage type to choose which takes like 2-4 minutes why would they ever buy bruise kits these pills for 600 and 1000 points because they are too lazy to interact with other department and are like RUSH RUSH POWERGAME LOOT FROM TENDRILS AND MEGAFAUNA

**Comment by Haevacht** [comment]
Rootbeer, let me give you some advice. Ignore Jughu. Entirely.

**Comment by jughu** [comment]
@haevacht poor guy didnt get his botany tools in his belt :^)

**Comment by Iamgoofball** [comment]
Ignore jughu he doesn't know how chem works

**Comment by Iamgoofball** [comment]
These pills are fine tbh, they don't healall and they still require advanced medical treatment

**Comment by jughu** [comment]
@iamgoofball im litteraly the one that suggested these chems

**Comment by SgtxRootbeer** [comment]
Well, at least I got the güfböl seal of approval.

**Comment by ExcessiveUseOfCobblestone** [comment]
I believe mentioning that taking another pill while basically having any noticeable amount of these chems in your body will cause you to OD.

**Comment by SgtxRootbeer** [comment]
The effects of OD'ing on this are so severe, and usage of it on lavaland, means you will be away from medical assistance. I felt that mentioning it was needed. If an overwhelming amount of people think that the desc should be changed, I don't mind. I just want this in as a concept.

**Comment by ExcessiveUseOfCobblestone** [comment]
I am pretty sure these chems will heal to full, just not as fast as healing from <50 to 50. May be wrong though

**Comment by SgtxRootbeer** [comment]
@ExcessiveUseOfCobblestone 
no, i just did the math, a 20 unit dose, will decay over the course of 100 ticks, healing at a rate of 0.5 units per tick, healing a total of 50 health, so if you have less then 50 brute _or_ burn you will not get healed fully.

@everyone
this is my first pr that i had to deal with involving balance, tell me, do you think i should revert this to bicaridine/kelotane, so it will heal you up to full, should i add 4 more units of each chemical, or should i leave as is?

**Comment by SgtxRootbeer** [comment]
now, the warning label for OD'ing is actually needed.

**Comment by SgtxRootbeer** [comment]
I did some research, and talked to a lot of my friends. i have come to the conclusion that using salicyclic acid and oxadrolone was a mistake.
that mistake has been reverted. apologies if this looks messy.

**Comment by KorPhaeron** [comment]
I don't think mining needs more healing supplies or options than they already have sorry

They already start with

-Medkits
-Medipens

And can easily obtain

-Brutekits
-More medipens
-Legion tumour things everywhere

Plus we have an entire two job slots dedicated to making pills and patches that nobody ever uses, miners can stop by chemistry if they want things like this

**Pull request closed without merging**